.github/workflows/auto-updates.yaml

```diff
@@ -154,17 +170,21 @@ jobs:
             console.info("Checksum file updated.");
             return true;
       - name: Create Pull Request
+        id: create-pr
         if: ${{ steps.update-checksum.outputs.result == 'true' }}
-        # V5 Beta needed because of https://github.com/peter-evans/create-pull-request/issues/1170
-        uses: peter-evans/create-pull-request@v5-beta
+        uses: peter-evans/create-pull-request@v4
         with:
           commit-message: Auto update Rust vendored sources checksums
           title: Auto update Rust vendored sources checksums
           labels: control, automated pr
           body: "[Auto-generated pull request](https://github.com/ubuntu/adsys/actions/workflows/auto-updates.yaml) by GitHub Action"
           branch: auto-update-rust-checksum
+          delete-branch: true
           token: ${{ secrets.GITHUB_TOKEN }}
-      - name: Push branch
-        if: ${{ steps.update-checksum.outputs.result == 'true' }}
-        run: |
-          git push origin auto-update-rust-checksum:main
+      - name: Automerge created Pull Request
```

Comment:
What does this gives us compared to only pushing the branch?

If I look at https://github.com/peter-evans/enable-pull-request-automerge#conditions, it seems to be really restrictive and moe complicated than a git push, I don’t see other simpification, hence my question.

Reply:
I'll resolve this since we already came to agreement on the right approach